cgmanifest.json

```diff
@@ -9901,8 +9901,8 @@
         "type": "other",
         "other": {
           "name": "libmbim",
-          "version": "1.26.2",
-          "downloadUrl": "https://www.freedesktop.org/software/libmbim/libmbim-1.26.2.tar.xz"
+          "version": "1.26.3",
+          "downloadUrl": "https://www.freedesktop.org/software/libmbim/libmbim-1.26.3.tar.xz"
```

Comment:
Looks unrelated- unfortunate find/replace accident, maybe?

```suggestion
          "version": "1.26.2",
          "downloadUrl": "https://www.freedesktop.org/software/libmbim/libmbim-1.26.2.tar.xz"
```

Reply:
The `libmbim` package was not modified, so we don't need to change the manifest entry for that package.

Reply:
The suggestion just rolls back the change to that entry

Reply:
oh ok got it, Yes this is mistake 
Thanks for pointing it out